hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/commit/TestUpsertPartitioner.java

```diff
@@ -434,6 +434,49 @@ public void testUpsertPartitionerWithSmallFileHandlingWithCanIndexLogFiles() thr
             "Insert should be assigned to fg1");
   }
 
+  @Test
+  public void testUpsertPartitionerWithSmallFileHandlingPickingMultipleCandidates() throws Exception {
+    final String partitionPath = DEFAULT_PARTITION_PATHS[0];
+
+    HoodieWriteConfig config =
```

Comment:
I will land this for now. But when you get a chance, can you make this test configurable w/ diff values for the config that we just added. Just to make sure the config is honored for diff values. just 2 diff values would suffice.

Reply:
@alexeykudinkin let's keep the code styling more consistent with whats there already? 

```
HoodieWriteConfig config = makeHoodieClientConfigBuilder()
```
instead of 

```
HoodieWriteConfig config =
                makeHoodieClientConfigBuilder
```